src/routes/tools/extreme-precipitation/index.svelte

```diff
@@ -0,0 +1,473 @@
+        logException(error);
+        notifier.error(
+          "Unable to Load Tool",
+          "Sorry! Something's probably wrong at our end. Try refereshing your browser. If you still see an error please contact us at support@cal-adapt.org.",
+          2000
+        );
+      });
+    window.scrollTo(0, 0);
+  });
+</script>
+
+<svelte:head>
+  <title>{tool.title}</title>
+  <link
+    href="https://api.mapbox.com/mapbox-gl-js/v2.0.1/mapbox-gl.css"
+    rel="stylesheet"
+  />
+</svelte:head>
+
+<Header
+  iconPaths="{tool.icons}"
+  title="{tool.title}"
+  description="An extreme weather event is an occurrence that is significantly different from typical weather at a specific location and time of year. Extreme precipitation events can lead to flooding, mudslides and other damaging events. In a changing climate the intensity and frequency of such events will likely change across California. This tool visualizes how climate models predict extreme precipitation events will change over the 21st century. "
+/>
+
+<ToolNavigation href="{`/tools/${tool.slug}`}" />
+
+<div id="explore-data" use:inview="{{}}" on:enter="{handleEntry}">
+  {#if appReady}
+    <ExploreData />
+  {:else}
+    <Loading />
+  {/if}
+</div>
+
+<div class="bx--grid">
+  <div id="about" use:inview="{{}}" on:enter="{handleEntry}">
+    <About
+      datasets="{datasets}"
+      on:datasetLoaded="{(e) => datasetStore.set(e.detail)}"
+    >
+      <div slot="description">
+        <p>
+          With this tool you can explore what an extreme precipitation event
+          looks like by providing estimates of intensity and frequency of
+          extreme precipitation events. The tools and visualizations allow you
+          to examine how extreme precipitation events are likely to change in a
+          warming climate over locations of interest to you.
+        </p>
+        <p>
+          By default, Cal-Adapt calculates extreme values of precipitation over
+          a 2-day period, and defines an extreme event as the lowest value from
+          Annual Maximum values in the historical period (1961–1990). Users can
+          override these defaults by selecting a new “event duration” (number of
+          days over which precipitation accumulates), or by selecting a
+          different “threshold“ value that corresponds to either the 90th, 95th
+          or 99th percentiles. The tool then displays the extreme events that
+          exceed the threshold in different ways. The Frequency chart shows the
+          estimated intensity of precipitation events (Return Level) for a
+          selected period (Return Period) and how it changes over the historical
+          period (1961–1990), mid-century (2035–2064) and end-century
+          (2071–2099). The other charts display the total number of events, the
+          timing of these events and the longest stretch of consecutive extreme
+          events.
+        </p>
+        <p class="h4">What is a Threshold value?</p>
```

Comment:
You could use `<h3>`s for these sub-headings as "About the Tool" is rendered as an `<h2>`.

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/186/commits/4663c64c03b58e1420dbd5f5154152ab169122c2